armonik/variables.tf

```diff
@@ -355,3 +355,9 @@ variable "metrics_server_chart_name" {
   type        = string
   default     = "metrics-server"
 }
+
+variable "environment_description" {
```

Comment:
My knowledge of terraform is limited but when I will deploy ArmoniK in test, is there a way to change these value?

Reply:
yes, in the `parameters.tfvars`. You can see an exemple in https://github.com/aneoconsulting/ArmoniK/pull/1161

Reply:
> And does the normalization indicated in this issue,[aneoconsulting/ArmoniK#1148](https://github.com/aneoconsulting/ArmoniK/issues/1148), will be in this infra or only in the documentation?

For now, no normalization is applied, but if any, I think it should be directly in ArmoniK